src/test/ui/associated-types/issue-90612.rs

```diff
@@ -0,0 +1,43 @@
+// check-pass
+
+#![feature(generic_associated_types)]
```

Comment:
Sorry, for the location I meant `src/test/ui/higher-rank-trait-bounds/normalize-under-binder`